src/components/Card/Card.tsx

```diff
@@ -28,22 +28,27 @@ import yoctosToUsdWithFallback from "@app/utils/yoctosToUsdWithFallback";
 import yoctosToNear from "@app/utils/yoctosToNear";
 import Image from "../mob.near/Image";
 import _address from "@app/utils/_address";
-import ModalDonation from "@app/modals/ModalDonation";
 
 const Card = (props: any) => {
-  const { payoutDetails } = props;
+  const { payoutDetails, allowDonate: _allowDonate } = props;
   const { potId } = useParams();
+
+  const { setDonationModalProps } = useContext<any>("donation-modal");
```

Comment:
Create a custom hook for this.

```ts
import { UserContextProps } from "@app/contexts/DonationModalProvider";

const useDonation = useContext<UserContextProps>("donation-modal");
export default useDonation;
```

```suggestion
  const { setDonationModalProps } = useDonation();
```